extensions/color-picker/src/utils.ts

```diff
@@ -1,6 +1,6 @@
 import { getPreferenceValues, Icon, Image, Keyboard, List } from "@raycast/api";
-import { HistoryColor, HistoryItem } from "./types";
 import ColorJS from "colorjs.io";
+import { HistoryColor, HistoryItem } from "./types";
```

Comment:
I saw so many unrelated changes on the PR. You should modify relevant code only.

Notes unrelated changes are not good for `git blame`.

Please revert all these unrelated changes. Thanks.

Reply:
This is result of VS Code `Organize Imports` command. It sorts imports and removes unused ones. I have seen a lot of extensions with unused imports which are basically dead code. I even think that this should be a formatting routine included in the CI.

Reply:
> By the way, you made the same change in your PR: [#13896 (files)](https://github.com/raycast/extensions/pull/13896/files)

I only changed those files in which I worked.

Reply:
Yes, but you didn't need to resort the imports to implement the feature your PR was intended for. It's the same here.